stories/Molecules/Text/Faq/Faq.jsx

```diff
@@ -7,26 +7,31 @@ import { P } from '../../../Atom/Base-typography/Paragraph/p';
 export const Faq = ({headerText, descriptionText}) => {
   useEffect(() => {
     FaqFun();
-  })
+  },[])
 
   return (
-   <div className="accordion" id="accordion">
-      <div className="accordion-head">
+   <ul class="accordion" id="accordion">
+      <li>
          <H5 label={headerText} />
-      </div>
-      <div className="accordion-body">
-         <P label={descriptionText} />
-         <P label={descriptionText} />
-         <P label={descriptionText} />
-      </div>
-      <div className="accordion-head">
+         <div class="accordion-panel">
+            <P label={descriptionText} />
+            <P label={descriptionText}  />
+         </div>
+      </li>
+      <li>
          <H5 label={headerText} />
-      </div>
-      <div className="accordion-body">
-         <P label={descriptionText} />
-         <P label={descriptionText} />
-         <P label={descriptionText} />
-      </div>
-   </div>
+         <div class="accordion-panel">
+            <P label={descriptionText} />
+            <P label={descriptionText} />
+         </div>
+      </li>
+      <li>
+         <H5 label={headerText} />
+         <div class="accordion-panel">
+            <P label={descriptionText} />
+            <P label={descriptionText} />
+         </div>
+      </li>
+   </ul>
```

Comment:
@harishkumar-srijan Add the classes on the li, h5 and add the styling on the classes instead of tags

Reply:
Done

Reply:
@harishkumar-srijan class names should be according to BEM

Reply:
Please BEM